src/rf/js/src/home/components/modals.js

```diff
@@ -293,8 +327,12 @@ var UploadModal = React.createBackboneClass({
             }
         }
 
-        if (!layerData.area) {
-            errors.area = ['Please enter an area.'];
+        if (layerData.capture_start && !layerData.capture_end) {
+            errors.capture_end.push('Please enter a date.');
+        }
+
+        if (layerData.capture_end && !layerData.capture_start) {
+            errors.capture_start.push('Please enter a date.');
         }
```

Comment:
Without a spec or idea of how the fields would be used, I erred towards the side of making things mandatory, but I should have done it the other way around.